libraries/Microsoft.Bot.Builder/Teams/TeamsInfo.cs

```diff
@@ -97,6 +161,16 @@ public static Task<TeamsChannelAccount> GetMemberAsync(ITurnContext turnContext,
             }
         }
 
+        /// <summary>
+        /// Generate the activity.
+        /// Support Both string LG result and structured LG result.
```

Comment:
This could use more clarity. Please add more information.

Reply:
updated.